assignments/array-methods.js

```diff
@@ -101,8 +101,13 @@ console.log(largeShirts);
 
 console.log('CHALLENGE 4 👇');
 
-// let ticketPriceTotal = [];
-// console.log(ticketPriceTotal);
+const donations = runners.map(runner => runner.donation);
+
+let ticketPriceTotal = donations.reduce((total, donation) => {
+    return total + donation;
+});
```

Comment:
Good job using reduce here. I do think though the challenge required just to use reduce but this is acceptable.